Welcome to some **Exceptions** in *python*
